.github/workflows/test_iree.yml

```diff
@@ -99,25 +100,19 @@ jobs:
       matrix:
         include:
           - name: cpu_llvm_task
-            runs-on: nodai-amdgpu-w7900-x86-64
+            runs-on: nodai-amdgpu-mi250-x86-64
```

Comment:
Looks like I'll need this change too. Might want to separate changes like these into their own PRs.